boards/rapidsilicon/virgo_tc_eval_soc_proto/board.cmake

```
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+# Note: Please change the --elf-file path,
+# server ip and port as per your machine.
+
+board_runner_args(
+                    openocd 
+                    "--cmd-load=load"
+                    "--use-elf"  
+                    "--elf-file=/home/junaid/zephyrproject/zephyr/build/zephyr/zephyr.elf" 
```

```suggestion
                    "--elf-file=/home/junaid/zephyrproject/zephyr/build/zephyr/zephyr.elf" 
```
Can we make this relative to the current path?

---

Done